### ISSUE TITLE
"Open terminal" isn't opening default terminal emulator

### ISSUE THREAD

**Issue by KaKi87** [Murmele/Gittyup#499]
Hello,

Although my distro is distributed with GNOME Terminal, I set [Tilix](https://github.com/gnunn1/tilix) as default terminal emulator (e.g. `x-terminal-emulator` opens it), yet, Gittyup's "open terminal" feature opens the former.

Thanks

**Comment by Murmele** [comment]
![image](https://user-images.githubusercontent.com/10099533/227779737-a4129ade-ed50-45c8-b77a-cf9143d29b4a.png)

You can manually setting a terminal. Just type in the terminal name

**Comment by KaKi87** [comment]
Mea culpa, Gittyup works perfectly.

My statement about `x-terminal-emulator` launching Tilix but Gittyup launching GNOME Terminal was false, because I tested the first half of it on one computer and the second half of it on another.

Turns out, the real issue is that Linux Mint's *Preferred Applications* feature (or `gsettings set org.cinnamon.desktop.default-applications.terminal exec tilix` doesn't affect `x-terminal-emulator`, so I had to run `sudo update-alternatives --config x-terminal-emulator` separately.

Thanks and sorry for the inconvenience.
